experimenter/experimenter/nimbus_ui_new/templates/experimenter_base.html

```diff
@@ -7,13 +7,14 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" href="{% static 'imgs/favicon.ico' %}" />
```

Comment:
Nit: shouldn't this use double quotes inside the templatetag?

Reply:
Good question, not sure if that's handled by the auto formatter?